packages/server/src/test/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnoweldgeControllerTests.java

```diff
@@ -54,13 +53,18 @@ public void equationsToModelRegNet() throws Exception {
 				.andExpect(status().isOk())
 				.andReturn();
 
-		Model amr = objectMapper.readValue(res.getResponse().getContentAsString(), Model.class);
-		log.info(amr.toString());
+		String responseContent = res.getResponse().getContentAsString(); // Remove double quotes
+		try {
+			final UUID regnetModelId = UUID.fromString(responseContent);
+			log.info(regnetModelId.toString());
+		} catch (final IllegalArgumentException e) {
+			log.error("Response content is not a valid UUID: " + responseContent);
+		}
```

Comment:
If @dvince2 or @kbirk could give me a hand onto why I get those errors on the test, it doesn't make sense to me?

```shell
2024-03-08 22:02:33 [INFO] REQUEST STARTED | KnowledgeController.equationsToModel | method=POST; uri=/knowledge/equations-to-model;  [software.uncharted.terarium.hmiserver.interceptors.RequestLoggingInterceptor:44]
2024-03-08 22:02:33 [DEBUG] Read "application/json" to [{"equations":["\\frac{d S}{d t} = -\\beta S I","\\frac{d I}{d t} = \\beta S I - \\gamma I","\\frac{d (truncated)...] [org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor:120]
2024-03-08 22:02:34 [DEBUG] Reading to [software.uncharted.terarium.hmiserver.models.dataservice.model.Model] [org.springframework.web.client.HttpMessageConverterExtractor:101]
2024-03-08 22:02:34 [INFO] Indexing: b69ebd82-1d0f-43e5-a84f-c5c6141ef298 into tds_model_tera_test_1.0 [software.uncharted.terarium.hmiserver.service.elasticsearch.ElasticsearchService:297]
2024-03-08 22:02:34 [DEBUG] Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/x-jackson-smile, application/cbor] [org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor:275]
2024-03-08 22:02:34 [DEBUG] Writing [b69ebd82-1d0f-43e5-a84f-c5c6141ef298] [org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor:120]
2024-03-08 22:02:34 [INFO] REQUEST COMPLETED | KnowledgeController.equationsToModel | 942 ms | method=POST; uri=/knowledge/equations-to-model;  [software.uncharted.terarium.hmiserver.interceptors.RequestLoggingInterceptor:77]
2024-03-08 22:02:34 [ERROR] Response content is not a valid UUID: "b69ebd82-1d0f-43e5-a84f-c5c6141ef298" [software.uncharted.terarium.hmiserver.controller.knowledge.KnoweldgeControllerTests:86]
```